src/Features/Core/Portable/UseThrowExpression/AbstractUseThrowExpressionDiagnosticAnalyzer.cs

```diff
@@ -33,12 +33,32 @@ namespace Microsoft.CodeAnalysis.UseThrowExpression
     internal abstract class AbstractUseThrowExpressionDiagnosticAnalyzer :
         AbstractBuiltInCodeStyleDiagnosticAnalyzer
     {
+        internal const string UseThrowExpressionFixableDiagnosticId = IDEDiagnosticIds.UseThrowExpressionDiagnosticId + "_Fixable";
+
+        private static readonly LocalizableString s_title = new LocalizableResourceString(nameof(FeaturesResources.Use_throw_expression), FeaturesResources.ResourceManager, typeof(FeaturesResources));
+        private static readonly LocalizableString s_message = new LocalizableResourceString(nameof(FeaturesResources.Null_check_can_be_simplified), FeaturesResources.ResourceManager, typeof(FeaturesResources));
+
+        private static readonly DiagnosticDescriptor s_descriptor = CreateDescriptorWithId(
+            IDEDiagnosticIds.UseThrowExpressionDiagnosticId,
+            s_title,
+            s_message);
+        private static readonly DiagnosticDescriptor s_fixableDescriptor = CreateDescriptorWithId(
```

Comment:
can you doc the difference between these two?

Reply:
note: this concern looks like it will go away once you address the above point :)